tools/downloader/converter.py

```diff
@@ -56,19 +56,28 @@ def prefixed_subprocess(prefix, args):
             sys.stdout.write(prefix + ': ' + line)
         return p.wait() == 0
 
+
 def convert_to_onnx(model, output_dir, args, stdout_prefix):
-    pytorch_converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
-    prefixed_printf(stdout_prefix, '========= {}Converting {} to ONNX',
-        '(DRY RUN) ' if args.dry_run else '', model.name)
-
-    pytorch_to_onnx_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
-                                                            dl_dir=args.download_dir / model.subdirectory)
-                            for arg in model.pytorch_to_onnx_args]
-    cmd = [str(args.python), str(pytorch_converter), *pytorch_to_onnx_args]
+    converters = {
+        'pytorch': Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
+    }
+    prefixed_printf(stdout_prefix, '========= {}Converting {} to ONNX from {}',
+                    '(DRY RUN) ' if args.dry_run else '', model.name, model.framework)
+
+    conversion_to_onnx_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
+                                                               dl_dir=args.download_dir / model.subdirectory)
+                               for arg in model.conversion_to_onnx_args]
+    converter = converters.get(model.framework)
+    if converter is None:
+        prefixed_printf(stdout_prefix, 'Conversion to ONNX not supported for {} framework', model.framework)
+        return False
```

Comment:
If we get here, it means that the model config is invalid. I think we should catch this error earlier; specifically, in `Model.deserialize`.

Reply:
Good point, thanks!